src/app/pcpolicy/pcpolicy.component.scss

```diff
@@ -0,0 +1,13 @@
+.pcpolicybody{
+	h1{
+		padding: 20px;
+	}
+}
+
+.mailpolicy{
```

Comment:
bootstrap has a [class](http://getbootstrap.com/css/#responsive-utilities-classes) for this